docs/install.md

```diff
@@ -11,11 +11,13 @@
 
         sudo apt-get install redis-server python3.4 python3.4-dev python-pip python-virtualenv virtualenvwrapper
 
-1. create a virtualenv:
```

Comment:
"1." is then incremented by Markdown on the fly, so keeping only "1." make it easier to maintain: if you need to insert a new item in the middle of the list, not risk to forget to increment all the other items ;)